WordPress/src/main/java/org/wordpress/android/ui/mysite/personalization/PersonalizationViewModel.kt

```diff
@@ -0,0 +1,174 @@
+package org.wordpress.android.ui.mysite.personalization
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.flow.firstOrNull
+import org.wordpress.android.R
+import org.wordpress.android.analytics.AnalyticsTracker.Stat
+import org.wordpress.android.fluxc.store.BloggingRemindersStore
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.ui.mysite.SelectedSiteRepository
+import org.wordpress.android.ui.mysite.cards.dashboard.posts.PostCardType
+import org.wordpress.android.ui.prefs.AppPrefsWrapper
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import org.wordpress.android.viewmodel.ScopedViewModel
+import javax.inject.Inject
+import javax.inject.Named
+
+const val CARD_TYPE_TRACK_PARAM = "type"
+
+@HiltViewModel
+class PersonalizationViewModel @Inject constructor(
+    @param:Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher,
+    private val appPrefsWrapper: AppPrefsWrapper,
+    private val selectedSiteRepository: SelectedSiteRepository,
+    private val bloggingRemindersStore: BloggingRemindersStore,
+    private val analyticsTrackerWrapper: AnalyticsTrackerWrapper
+) : ScopedViewModel(bgDispatcher) {
+    private val _uiState = MutableLiveData<List<DashboardCardState>>()
+    val uiState: LiveData<List<DashboardCardState>> = _uiState
+
+    fun start() {
```

Comment:
For some reason this start function in ViewModel was used in the past.  Not sure, if it is needed anymore.  We could probably use init() instead, then don't have to have call this in onCreate in Activity explicitly.

Reply:
Hey @ravishanker 
I initially used the `init()` function, but on test functions, it was becoming harder to mock all the items for a test. So if we use the `init` block, I have to either set up all the mocks in the @Before function of tests or initialize ViewModel only in tests.  Both solutions didn't look right to me somehow, hence resorted to using `start()`. 

If you have any suggestions, I am all ears.